bespin_api_v2/api.py

```diff
@@ -0,0 +1,97 @@
+import json
+from rest_framework import viewsets, permissions, status, mixins
+from rest_framework.response import Response
+from rest_framework.decorators import detail_route
+from django.db import transaction
+from bespin_api_v2.serializers import AdminWorkflowSerializer, AdminWorkflowVersionSerializer, VMStrategySerializer, \
+    WorkflowConfigurationSerializer
+from data.serializers import JobSerializer
+from data.models import Workflow, WorkflowVersion, VMStrategy, WorkflowConfiguration, JobFileStageGroup
+from data.jobfactory import create_job_factory_for_workflow_configuration, JobOrderData
+from data.exceptions import BespinAPIException
+
+
+class CreateOnlyViewSet(mixins.CreateModelMixin,
```

Comment:
This is `CreateOnly` but in addition to Create, it also mixes in List and Retrieve. I suppose the idea is to prevent delete and update, but don't think that's entirely clear.